src/poprox_storage/repositories/data_stores/db.py

```diff
@@ -32,48 +32,54 @@ def _id_query(self, query):
         result = self.conn.execute(query).fetchall()
         return [row[0] for row in result]
 
+    def _insert_model(
+        self,
+        table_name: str,
+        model,
+        addl_fields: dict[str, Any] | None = None,
+        *,
+        constraint: str | None = None,
+        exclude=None,
+        commit: bool = True,
+    ):
+        fields: dict[str, Any] = model.model_dump(exclude=exclude)
+
+        if addl_fields:
+            fields.update(addl_fields)
```

Comment:
For any fields that have different names from the database columns, we can use `exclude` to omit them from the insert and `addl_fields` to add them back in with a different name